.github/ISSUE_TEMPLATE/docs_report.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
might as well chane this one too to be `.yml`

Reply:
Sure, done

Reply:
Actually, it seems like you need to have a form if it's `.yml`, so changing it back to `.md`.

Reply:
yeah it can be a simple form with one textarea but doesn't matter tbh.